packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Link/index.js

```diff
@@ -44,6 +44,11 @@ class LinkContainer extends Component {
     }
   }
 
+  componentWillUnmount () {
```

Comment:
reset the component UI on unmount and wipe the bank accounts from the redux state. We do this because we lose the token that's needed for linking a bank when the component's state is destroyed.